modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/menu/MenuProvider.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayDropDown from '@clayui/drop-down';
+import React, {useEffect, useMemo, useRef, useState} from 'react';
+
+import AccountMenuContent from './AccountMenuContent';
+import OrganizationMenuContent from './OrganizationMenuContent';
+import UserMenuContent from './UserMenuContent';
+
+const content = {
+	account: AccountMenuContent,
+	organization: OrganizationMenuContent,
+	user: UserMenuContent,
+};
+
+export default function MenuProvider({
+	alignElementRef,
+	data: propData,
+	parentData,
+}) {
+	const [active, setActive] = useState(false);
+	const [data, updateData] = useState(false);
+	const menuRef = useRef(null);
+
+	/**
+	 * the useEffect below force the component repositioning
+	 */
```

Comment:
We don't want comments

Reply:
@kresimir-coko this sort of comment should be okay, as it seems to be explaining why we would be using this useEffect. I could imagine someone refactoring this in the future an accidentally causing a regressing if they didn't know why there were two separate effects.

[Per Guidelines](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/comments.md#guidelines)
> For cases where you want to explain some internal detail of the code (to help the reader understand how it works, but where such explanation wouldn't really belong in any extracted "documentation")

Reply:
I'll update it in a single line comment starting with capital letter ;)